### PR TITLE
Update enable-exploit-protection.md

### LINKED ISSUE #11835: Incorrect config steps
[
https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/enable-exploit-protection?view=o365-worldwide#intune

Step#2 >> Devices >> Configuration Profiles >> Create Profile

Step#3 >> Profile Type - Templates

Template Name - Endpoint Protection

image

]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 506a8799-0002-3db2-215d-7fdbb679619d
* Version Independent ID: 312dc4ff-d20c-fb66-01d7-bc14170ae5a4
* Content: [Turn on exploit protection to help mitigate against attacks](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/enable-exploit-protection?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/enable-exploit-protection.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/enable-exploit-protection.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to step2 and 3

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11835

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/enable-exploit-protection.md [hunk 1]
```diff
@@ -114,9 +114,9 @@ The result is that DEP is enabled for *test.exe*. DEP won't be enabled for any o
 
 1. Sign in to the [Azure portal](https://portal.azure.com) and open Intune.
 
-2. Go to **Device configuration** \> **Profiles** \> **Create profile**.
+2. Go to **Device configuration** \> **Configuration Profiles** \> **Create profile**.
 
-3. Name the profile, choose **Windows 10 and later** and **Endpoint protection**.
+3. Name the profile, choose **Windows 10 and later**, Select **templates** for Profile type and choose **Endpoint protection** under template name.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-exploit-protection.md]:
```suggestion
3. Name the profile, choose **Windows 10 and later**, select **templates** for Profile type and choose **Endpoint protection** under template name.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**